runtime/runtime/src/actions.rs

```diff
@@ -96,6 +96,8 @@ pub(crate) fn execute_function_call(
         random_seed,
         view_config,
         output_data_receivers,
+        current_gas_price: apply_state.gas_price,
+        receipt_gas_price: action_receipt.gas_price,
```

Comment:
nit: are we worried about someone swapping these around accidentally in the future as they have the same types?  In the past, I've tried the following two mechanisms to safeguard against such accidents:
- If one value is guaranteed to be an integer multiple of the other, store the first value as the original type and for the second we store the multipler.  I don't think this is applicable here.
- Introduce a `new` function that can add some checks e.g. if one value should be `>=` than the other, etc. and simply panic if invariants are broken.

No strong opinion about this, just thinking out loud.

Reply:
I don't think even `>=` is guaranteed to hold here. And IMO, accidentally swapping two field values must absolutely be detected by our test infrastructure, or else our testing strategy is not fit for the purpose. In this case, the integration tests would fail because the host functions return the wrong values.